src/ray/core_worker/transport/direct_actor_transport.h

```diff
@@ -522,7 +579,9 @@ class CoreWorkerDirectTaskReceiver {
   std::shared_ptr<DependencyWaiter> waiter_;
   /// Queue of pending requests per actor handle.
   /// TODO(ekl) GC these queues once the handle is no longer active.
-  std::unordered_map<WorkerID, SchedulingQueue> scheduling_queue_;
+  std::unordered_map<WorkerID, SchedulingQueue*> actor_scheduling_queues_;
+  // Queue of pending normal (non-actor) tasks.
+  SchedulingQueue *normal_scheduling_queue_ = new NormalSchedulingQueue();
```

Comment:
Could you make this and the actor_scheduling_queues_ value type a unique_ptr<SchedulingQueue>? We try not to use raw pointers directly.